tests/phpunit/AutoReview/ProjectCode/ProjectCodeProvider.php

```diff
@@ -98,7 +97,6 @@ final class ProjectCodeProvider
         ComposerExecutableFinder::class,
         BadgeApiClient::class,
         ResultsLoggerTypes::class,
-        MutantCreatingConsoleLoggerSubscriber::class,
```

Comment:
Why did we remove it?

Reply:
We're not logging these events any more because we create mutants as we go in either mode.

Reply:
I also removed MutantsCreationWasFinished, MutantsCreationWasStarted

Reply:
And `MutantWasCreated`